src/launchpad/cli.py

```diff
@@ -411,6 +409,9 @@ def _format_bytes(size: int) -> str:
     return f"{size_float:.1f} TB"
 
 
+cli.add_command(size_command)
```

Comment:
Seems we should cut out the existing CLI `android` && `ios` subcommands in favor of this new `size` command?